core/src/overture/make.bat

```diff
@@ -28,21 +28,35 @@ MKDIR %TARGET%\armeabi-v7a>nul 2>nul
 MKDIR %TARGET%\x86>nul 2>nul 
 MKDIR %TARGET%\arm64-v8a>nul 2>nul 
 
+SET CC=%ANDROID_NDK_HOME%\toolchains\arm-linux-androideabi-4.9\prebuilt\windows-x86_64\bin\arm-linux-androideabi-gcc.exe
```

Comment:
It seems you are using the outdated ndk

Reply:
In fact, the ndk I used is not outdated.
But I found that maybe could be written like this:
`SET CC=%ANDROID_ARM_TOOLCHAIN%\bin\arm-linux-androideabi-gcc.exe`

Reply:
I think this one is better. Please update the path here.